build.yaml

```diff
@@ -22,8 +22,14 @@ cassandra:
   - 3.0
   - '3.11'
 os:
-  - ubuntu/trusty64
+  - ubuntu/bionic64/ruby-driver
 build:
+  - script: |
+      ruby_version=`ruby --version`
+      if [[ `ruby --version` == jruby* ]]; then
+        # Remove gemfile lock for jruby as build is different
+        rm Gemfile.lock
```

Comment:
This isn't required for older versions of bundler, but for new versions presence of `Gemfile.lock` causes `bundle install` to fail with jruby as the generated version is different (`3.2.3.rc.1-java` instead of `3.2.3.rc.1`) for example.